src/index.html

```diff
@@ -17,6 +17,93 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#apple"
+              class="is-active nav_link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#samsung"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#smartphones"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#laptops&computers"
```

Comment:
Don't use spaces in links. The `href` attribute value `#laptops&computers` should be `#laptops-computers` or another valid identifier without spaces.

Reply:
❌